spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap

```diff
@@ -135,7 +135,7 @@ Array [
                     Total Household Size
                   </h4>
                   <p>
-                    1
+                    2
```

Comment:
was this incorrect before? or unintentional

Reply:
it was incorrect before